src/sass/components/_repo-images.scss

```diff
@@ -0,0 +1,42 @@
+// Repo Images
+
+// Variables specific to repo images
+$repo-img-size: calc(3.5rem + 3vw);
+$repo-img-size-static: 4.75rem;
+$repo-img-pos-adjust: -1.125rem;
+
+.ms-repo-item {
+  // increase default height/weight of icon in repo visualpicker
+  .slds-media__figure {
+    > div {
+      height: $repo-img-size;
+      width: $repo-img-size;
+
+      .slds-avatar_large {
```

Comment:
Does this need to be nested so deeply for specificity reasons? Seems like it could just be nested under `.ms-repo-item`?

Reply:
nope, I'll un-nest.